site/en/docs/extensions/mv3/devtools/index.md

```diff
@@ -138,15 +137,17 @@ components of a DevTools extension.
 
 ### Injecting a content script {: #injecting }
 
-The DevTools page can't call [`scripting.executeScript`][17] directly. To inject a content script from
-the DevTools page, you must retrieve the ID of the inspected window's tab using the
-[`inspectedWindow.tabId`][18] property and send a message to the background page. From the
-background page, call [`scripting.executeScript`][19] to inject the script.
+The DevTools page can't call [`scripting.executeScript`][api-scripting-executescript] directly. To
```

Comment:
```suggestion
The DevTools page can't call [`scripting.executeScript()`][api-scripting-executescript] directly. To
```

Reply:
Methods should have '()' after them. Please make sure this is consistent throughout.